### PR TITLE
Issue#SC-1315: api doc for bulk user upload (shadow db)

### PR DESCRIPTION
**Pull request by aishwa8141**

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://493.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://493.qa.docs.sunbird.org/2.3.0/apis/bulkupload// <br />  <br/> **Modified sections are -** <br />

**Comment by manzarul** [comment]
PR for release-2.4.0

**Comment by keshavprasadms** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://493.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://493.qa.docs.sunbird.org/2.3.0/apis/bulkupload// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://493.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://493.qa.docs.sunbird.org/2.3.0/apis/bulkupload// <br />  <br/> **Modified sections are -** <br />

File: apis/bulkupload/bulkserviceapi.yaml [hunk 1]
```diff
@@ -412,6 +412,71 @@ paths:
...
+          format: file
+          description: >-
+            This is a .csv file.
+            - Each line of the file is a data record. Each record consists of one or more user fields, separated by commas. Each successfully created user is associated with root org of the uploader (org admin).
+
+            The CSV file format supported is as follows:
+              - Name: Name of user as per state records (mandatory)
+              - Email: Email. Either phone or email is mandatory.
+              - Phone: Phone number. Either phone or email is mandatory.
+              - External Org ID: School ID as provided in Diksha .
```
Comment (Sunitashelar) [inline apis/bulkupload/bulkserviceapi.yaml]: @aishwa8141 @manzarul as discussed, please check these descriptions as we cannot use Diksha etc in Sunbird. If we make it generic, then we can add it to Sunbird documentation. Else, we will need to check where to keep this file.

File: apis/bulkupload/bulkserviceapi.yaml [hunk 2]
```diff
@@ -412,6 +412,71 @@ paths:
...
+              - External User ID: State ID of the user (mandatory)
```
Comment (Sunitashelar) [inline apis/bulkupload/bulkserviceapi.yaml]: @aishwa8141 @manzarul  Similar to comment above. Cannot have State ID in a generic scenario.

**Comment by Sunitashelar** [comment]
Closing the request as its outdated

**Pull request closed without merging**